mne/preprocessing/stim.py

```diff
@@ -67,3 +72,139 @@ def eliminate_stim_artifact(raw, events, event_id, tmin=-0.005,
         elif mode == 'window':
             raw[picks, first_samp:last_samp] = data * window[np.newaxis, :]
     return raw
+
+
+def fix_stim_artifact_raw(raw, events, event_id, tmin=-0.005,
```

Comment:
actually I am wondering if this should be merged with the other function `fix_stim_artifact`. It looks a bit odd to me to have one function for `raw` and another for `epochs` or `evoked`